<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shoudongceshi</class>
 <widget class="QWidget" name="shoudongceshi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>718</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>手动测试</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/image/Chart/logo2.ico</normaloff>:/image/Chart/logo2.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_11">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>1005</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/image/Chart/gdoushoudong.png);
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="4,1">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>CAMERA</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="minimumSize">
              <size>
               <width>640</width>
               <height>480</height>
              </size>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOn</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOn</enum>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>756</width>
                <height>540</height>
               </rect>
              </property>
              <property name="minimumSize">
               <size>
                <width>640</width>
                <height>480</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="camaraget" name="lb_camera">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>IMAGE</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QScrollArea" name="scrollArea_3">
               <property name="minimumSize">
                <size>
                 <width>640</width>
                 <height>480</height>
                </size>
               </property>
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents_3">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>771</width>
                  <height>499</height>
                 </rect>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>640</width>
                  <height>480</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_15">
                 <item>
                  <widget class="QLabel" name="lb_image">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="font">
                    <font>
                     <family>DejaVu Sans Mono</family>
                    </font>
                   </property>
                   <property name="text">
                    <string>当前检测数量</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Elapsed Time(ms)</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QLineEdit" name="le_currentnum">
                   <property name="maximumSize">
                    <size>
                     <width>400</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="le_usedtime">
                   <property name="maximumSize">
                    <size>
                     <width>400</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QPushButton" name="pb_fangda">
                   <property name="minimumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="res.qrc">
                     <normaloff>:/image/Chart/Plus.png</normaloff>:/image/Chart/Plus.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pb_suoxiao">
                   <property name="minimumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="res.qrc">
                     <normaloff>:/image/Chart/Minus.png</normaloff>:/image/Chart/Minus.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>TEMPLATE</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QScrollArea" name="scrollArea_2">
               <property name="minimumSize">
                <size>
                 <width>640</width>
                 <height>480</height>
                </size>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOn</enum>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOn</enum>
               </property>
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents_2">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>754</width>
                  <height>482</height>
                 </rect>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>640</width>
                  <height>480</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_12">
                 <item>
                  <widget class="QLabel" name="lb_template">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1,0,1">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Number</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_3">
                   <property name="text">
                    <string>1</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Minimum</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_4">
                   <property name="text">
                    <string>750</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBox">
                 <property name="singleStep">
                  <double>0.010000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.110000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox">
                 <property name="currentText">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pb_qutu">
           <property name="font">
            <font>
             <family>DejaVu Sans Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>取图</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/image/Chart/qutu.png</normaloff>:/image/Chart/qutu.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_cuangjian">
           <property name="font">
            <font>
             <family>DejaVu Sans Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>创建模块</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/image/Chart/chuangjian.png</normaloff>:/image/Chart/chuangjian.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_delete">
           <property name="font">
            <font>
             <family>DejaVu Sans Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>删除模块</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/image/Chart/shanchu.png</normaloff>:/image/Chart/shanchu.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_test">
           <property name="font">
            <font>
             <family>DejaVu Sans Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>测试</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/image/Chart/ceshi.png</normaloff>:/image/Chart/ceshi.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_save">
           <property name="font">
            <font>
             <family>DejaVu Sans Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>保存</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/image/Chart/save.png</normaloff>:/image/Chart/save.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_quit">
           <property name="font">
            <font>
             <family>DejaVu Sans Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>退出</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/image/Chart/tuichu.png</normaloff>:/image/Chart/tuichu.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>camaraget</class>
   <extends>QLabel</extends>
   <header>camaraget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
